test/OrchardCore.Tests/Recipes/RecipeExecutorTests.cs

```diff
@@ -32,12 +32,17 @@ public async Task ShouldTrimValidScriptExpression(string recipeName, string expe
 
                 var recipeEventHandlers = new List<IRecipeEventHandler> { new RecipeEventHandler() };
                 var loggerMock = new Mock<ILogger<RecipeExecutor>>();
+                var localizerMock = new Mock<IStringLocalizer<RecipeExecutor>>();
+
+                localizerMock.Setup(localizer => localizer[It.IsAny<string>()])
+                .Returns((string name) => new LocalizedString(name, name));
```

Comment:
```suggestion
                    .Returns((string name) => new LocalizedString(name, name));
```

Reply:
This is an extra space. I rather leave it where VS place it

Reply:
If I'm not wrong VS should place it after 4 spaces, this is a quick example

![Screenshot 2024-05-25 065208](https://github.com/OrchardCMS/OrchardCore/assets/3237266/b1917fdf-12dc-4274-ace8-174b66e34389)

Reply:
> If I'm not wrong VS should place it after 4 spaces, this is a quick example .

How is this formatted? Even if I use the latest version of vs , it doesn't format this style of code automatically! 😳

Reply:
Unless there's a change in `.editorconfig` file

Reply:
@hishamco feel free to check out the branch and try it/commit if VS formats it different.

If you want to try this out, just hit enter after the end of the line and do not add any space/tab at the beginning